src/BootstrapTable.js

```diff
@@ -421,7 +420,7 @@ class BootstrapTable extends Component {
       expandColumnOptions.expandColumnBeforeSelectColumn = true;
     }
     const colGroups = Util.renderColGroup(columns, this.props.selectRow, expandColumnOptions, this.props.version);
-    const tableFooter = this.renderTableFooter(footerData, footerFormatterReturnData, columns, colGroups);
+    const tableFooter = this.renderTableFooter(this.props.footerData, this.state.data, columns, colGroups);
```

Comment:
@AllenFang 
As I removed `footerFormatterData` from `options`, I'm using `this.state.data` to send back data to footerdata formatter.